app/content/profile/index.js

```diff
@@ -340,7 +353,7 @@ function Index() {
                         userId,
```

Comment:
Should we be doing try / catch here? Maybe make a handler function? Also goes for similar calls to handleImageUpload in team index page (team image upload).

The hooks/imageUpload.js file (where the handleImageUpload function is defined) also needs checking to see if it has enough try catches.